conductor_api/src/interface.rs

```diff
@@ -1055,7 +1072,58 @@ pub mod tests {
                     "instance_id" : "test-instance-1",
                     "zome" : "greeter",
                     "function" : "hello",
+                })),
+            ))
+            .expect("Invalid call to handler");
+        assert_eq!(
+            response_str,
+            r#"{"jsonrpc":"2.0","error":{"code":-32602,"message":"Holochain Instance Error: Holochain instance is not active yet."},"id":"0"}"#
+        );
 
+        let response_str = handler
+            .handle_request_sync(&create_call_str(
+                "call",
+                Some(json!({
+                    "instance_id" : "test-instance-1",
+                    "zome" : "greeter",
+                    "function" : "hello",
+                    "token" : "bogus token",
+                })),
+            ))
+            .expect("Invalid call to handler");
+        assert_eq!(
+            response_str,
+            r#"{"jsonrpc":"2.0","error":{"code":-32602,"message":"Holochain Instance Error: Holochain instance is not active yet."},"id":"0"}"#
+        );
+
+        let response_str = handler
+            .handle_request_sync(&create_call_str(
+                "call",
+                Some(json!({
+                    "instance_id" : "test-instance-1",
+                    "zome" : "greeter",
+                    "function" : "hello",
+                    "provenance" : {"bad_provenance" : "bogus"},
+                })),
+            ))
+            .expect("Invalid call to handler");
+        assert_eq!(
+            response_str,
+            r#"{"jsonrpc":"2.0","error":{"code":-32602,"message":"invalid provenance: missing field `source` at line 1 column 26"},"id":"0"}"#
+        );
+
+        let response_str = handler
+            .handle_request_sync(&create_call_str(
+                "call",
+                Some(json!({
+                    "instance_id" : "test-instance-1",
+                    "zome" : "greeter",
+                    "function" : "hello",
+                    "token" : "bogus token",
+                    "provenance" : {
+                        "source" : "some_source",
+                        "signature" : "some_signature",
+                    },
```

Comment:
I was wondering why we see the "instance not active yet" error below, for this provenance. The answer is because the provenance check happens within the instance but in this test we don't have an instance, right?

Reply:
yep.  I wish we did, and then it could be a better test, but this just tests that the routes are in place.